src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/index.tsx

```diff
@@ -9,67 +9,26 @@ import {
 	MdxTab,
 	MdxTabs,
 	MdxTable,
+	MdxH1,
+	MdxH2,
+	MdxH3,
+	MdxH4,
+	MdxH5,
+	MdxH6,
 } from 'components/dev-dot-content/mdx-components'
 import DocsAnchor from 'components/docs-anchor'
-import Heading from 'components/heading'
 import Text from 'components/text'
 import Image from 'components/image'
 import devDotStyles from 'components/dev-dot-content/dev-dot-content.module.css'
 import { ImageProps } from 'components/image/types'
 
-/**
- * Used by `makeHeadingElement`.
- */
-const HEADING_LEVELS_TO_PROPS = {
-	1: {
-		size: 500,
-		weight: 'bold',
-	},
-	2: {
-		size: 400,
-		weight: 'bold',
-	},
-	3: {
-		size: 300,
-		weight: 'bold',
-	},
-	4: {
-		size: 200,
-		weight: 'semibold',
-	},
-	5: {
-		size: 200,
-		weight: 'semibold',
-	},
-	6: {
-		size: 200,
-		weight: 'semibold',
-	},
-}
-
 // This function returns a simple object containing the default components
 // The `additionalComponents` param is purely for convenience.
 // It is intended for use with `next-mdx-remote`.
 export default function defaultMdxComponents({ additionalComponents = {} }) {
 	return Object.assign(_defaultComponents(), additionalComponents)
 }
 
-/**
- * Returns the Heading element with its given props applied and custom props we
- * apply for styling these components in docs pages.
- */
-export function makeHeadingElement(level, props) {
```

Comment:
In moving shared MDX heading utilities to `components`, we're starting to move stuff that is used across `docs` and `tutorial` content out of `layouts/sidebar-sidecar`.